src/rpc/dial.rs

```diff
@@ -467,6 +467,7 @@ impl DialBuilder<WithoutCredentials> {
         mdns_uri: Option<Parts>,
         mut original_uri_parts: Parts,
     ) -> Result<ViamChannel> {
+        log::debug!("Dialing");
```

Comment:
I would re-import the string.rs if this is possible otherwise I am concern with people accidentally remove/changing them

Reply:
if the DIALDBG_DEVELOPMENT environment variable is accessible would wrap to log statements so they are only written under test and it's clear in the code what they are meant for

Reply:
> I would re-import the string.rs if this is possible otherwise I am concern with people accidentally remove/changing them

Good idea; will do.

> if the DIALDBG_DEVELOPMENT environment variable is accessible would wrap to log statements so they are only written under test and it's clear in the code what they are meant for

I'm a little lost on what you're suggesting here. To only `log::debug` if `DIALDBG_DEVELOPMENT` is set?

Reply:
> > I would re-import the string.rs if this is possible otherwise I am concern with people accidentally remove/changing them
> 
> Good idea; will do.
> 
> > if the DIALDBG_DEVELOPMENT environment variable is accessible would wrap to log statements so they are only written under test and it's clear in the code what they are meant for
> 
> I'm a little lost on what you're suggesting here. To only `log::debug` if `DIALDBG_DEVELOPMENT` is set?

I was thinking of a macro looking like this : 
`
macro_rules! dbg_log {

    ($($args: tt)*) => {
        if std::env::var_os("DIALDBG").is_some(){
            log::debug!($($args)*);
        }

    }
}
`

Reply:
I like the idea of using a macro, but my idea was that `DIALDBG_DEVELOPMENT` was to be used to print rust-utils' logs to dialdbg's output, so that I could see what rust-utils was outputting during development of dialdbg. What you've described above seems like it'll log rust-utils' logs only if `DIALDBG_DEVELOPMENT` is defined. I _could_ restrict rust-utils' logs (or a subset of them) to only `log::debug!` if some environment variable is set, but that feels strange. In my head, the debug messages I've added, like "Dialing", were no different from the existing, "normal" log statements.

Reply:
I see I didn't realize we would have the logs on most (all) the time make sense to not wrap them then